### PR TITLE
Add EdgeDetection Post Process Page

### PR DESCRIPTION
**Pull request by UN1TOne**
Add EdgeDetection Post Process Page.

It's about the contents of "EdgeDetection Post Process"

[](https://github.com/BabylonJS/Babylon.js/pull/15721)

### CODE REVIEW

File: content/toolsAndResources/assetLibraries/postProcessLibrary/edgeDetectionPP.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+// Creates the post process
+var edgeDetectionPostProcess = new BABYLON.EdgeDetectionPostProcess("EdgeDetection", camera);
+```
+<Playground id="#HBSUDW" title="Edge Detection Post Process Demo" description="Edge Detection Post Process Demo"/>
+
+## Going further
+
+You can adjust the thickness of the outlines like so:
+
+<!-- default 값 변경 필요 -->
```
Comment (RaananW) [inline content/toolsAndResources/assetLibraries/postProcessLibrary/edgeDetectionPP.md]: please remove any comments that are not in english

**Review by RaananW — CHANGES_REQUESTED** [review]
In order to see the page it must be added to the structure.json file in the right place. The file is pretty self-explanatory, but I will be able to help if any questions arise. Just be sure to add it to the right parent.

Otherwise LGTM after removing the non-english comments in the page.

Also be sure to add the language used after the \``` (i.e. ```javascript)

File: content/toolsAndResources/assetLibraries/postProcessLibrary/edgeDetectionPP.md (hunk 1 above)
Comment (UN1TOne) [reply content/toolsAndResources/assetLibraries/postProcessLibrary/edgeDetectionPP.md]: Oops I'm sorry for that

**Review by RaananW — APPROVED** [review]

**Comment by RaananW** [comment]
Please ignore the CI errors :-) I am working on fixing it

**Comment by bjsplat** [comment]
snapshot for this PR available at:

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/DOCS/refs/pull/1224/merge

**Comment by RaananW** [comment]
LGTM! https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/DOCS/refs/pull/1224/merge/toolsAndResources/assetLibraries/postProcessLibrary/edgeDetectionPP/ :-)

Merging.

**Pull request merged**